main/nic-pipnone.json

```diff
@@ -39,8 +39,7 @@
         ],
         "networkSecurityGroup": {
                 "id": "[parameters('networkSecurityGroupId')]"
-        },
-        "enableIPForwarding":true
+        }
```

Comment:
why we removed these?

Reply:
After discussion, It is not required in NIOS and is not impacting the NIOS VM setup.